src/vhip/mlmodel/read_sequence.py

```diff
@@ -38,3 +38,18 @@ def read_headers(path: str) -> list[str]:
     for record in SeqIO.parse(path, "fasta"):  # pyright: ignore[reportUnknownVariableType, reportUnknownMemberType]
         result.append(record.id)  # pyright: ignore[reportUnknownMemberType]
     return result  # pyright: ignore[reportUnknownVariableType]
+
+
+def read_gene_products(path: str) -> list[str]:
```

Comment:
can you change function name to read_gene_content

Reply:
Or what is the reason you're using the word product?

Reply:
So the header has both an ID and a product like protein product like beta sliding clamp etc. read_headers already returns just the ID (i.e. ABDEAL_00005). So I made this function to extract the other part of the header. Really, I feel like read_headers should be changed to read_id or something like that. Read_headers works the same for genome fasta files, extracting only the first part (the ID).